client/performance-profiler/utils/metrics.ts

```diff
@@ -0,0 +1,127 @@
+import { translate } from 'i18n-calypso';
+import { PerformanceMetrics, Valuation } from '../types/performance-metrics';
+
+export const metricsNames = {
+	fcp: { displayName: 'Loading speed', name: 'First Contentful Paint' },
+	lcp: {
+		displayName: 'Largest content load',
+		name: 'Largest Contentful Paint',
+	},
+	cls: {
+		displayName: 'Visual stability',
+		name: 'Cumulative Layout Shift',
+	},
+	inp: { displayName: 'Interactivity', name: 'Interaction to Next Paint' },
+	ttfb: { displayName: 'Server responsiveness', name: 'Time to First Byte' },
+};
+
+export const metricValuations = {
+	fcp: {
+		good: translate( "Your site's loading speed is good" ),
+		needsImprovement: translate( "Your site's loading speed is moderate" ),
+		bad: translate( "Your site's loading speed needs improvement" ),
+		heading: translate( 'What is loading speed?' ),
+		aka: translate( '(aka First Contentful Paint)' ),
+		explanation: translate(
+			'Loading speed reflects the time it takes to display the first text or image to visitors. The best sites load in under 1.8 seconds.'
+		),
+	},
+	lcp: {
+		good: translate( "Your site's largest content load is good" ),
+		needsImprovement: translate( "Your site's largest content load is moderate" ),
+		bad: translate( "Your site's largest content load needs improvement" ),
+		heading: translate( 'What is largest content load?' ),
+		aka: translate( '(aka Largest Contentful Paint)' ),
+		explanation: translate(
+			'Largest content load measures the time it takes for the largest visible element (like an image or text block) on a page to load. The best sites load in under 2.5 seconds.'
+		),
+	},
+	cls: {
+		good: translate( "Your site's visual stability is good" ),
+		needsImprovement: translate( "Your site's visual stability is moderate" ),
+		bad: translate( "Your site's visual stability needs improvement" ),
+		heading: translate( 'What is visual stability needs?' ),
+		aka: translate( '(aka Content Layout Shift)' ),
```

Comment:
Is this supposed to be _Cumulative_ Layout Shift?

Reply:
Yes it's supposed to be `Cumulative Layout Shift`.